src/System.Data.Odbc/src/System.Data.Odbc.csproj

```diff
@@ -111,13 +111,19 @@
     <Compile Include="Common\System\Data\Common\UnsafeNativeMethods.cs" />
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetsLinux)' == 'true' And '$(IsPartialFacadeAssembly)' != 'true' ">
-    <Compile Include="Common\System\Data\Common\ExternDll.Linux.cs" />
+    <Compile Include="$(CommonPath)\Interop\Linux\Interop.Libraries.cs">
+      <Link>Common\Interop\Linux\Interop.Libraries.cs</Link>
+    </Compile>
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetsOsx)' == 'true' And '$(IsPartialFacadeAssembly)' != 'true' ">
-    <Compile Include="Common\System\Data\Common\ExternDll.Osx.cs" />
+    <Compile Include="$(CommonPath)\Interop\OSX\Interop.Libraries.cs">
```

Comment:
Once you unify the change into the Unix file, you can delete both ItemGroups the TargetsOsx and TargetsLinux ones and then only use one with TargetsUnix and include the file there, i.e: 

```xml
<ItemGroup Condition=" '$(TargetsUnix)' == 'true' ">
   <Compile Include="$(CommonPath)\Interop\Windows\Interop.Libraries.cs">
       <Link>Common\Interop\Windows\Interop.Libraries.cs</Link>
    </Compile>
</ItemGroup>
```

Also, `IsPartialFacadeAssembly` is only set to true when `TargetGroup` == `netfx` which can't be built on non windows environments, so it wouldn't be needed for this `ItemGroup` as a condition.